benchmark_test.rb

```diff
@@ -0,0 +1,74 @@
+# Run 'ruby -I . -I lib/ benchmark_test.rb'
+require 'benchmark/ips'
+require 'mongoid'
+require_relative 'lib/active_model_serializers_cancancan'
+
+require 'rspec/its'
+
+Mongoid.configure do |config|
```

Comment:
can we use postgresql please since this is a database we all use?

Reply:
maybe `sqlite3` with inmemory db?? :D